.pipelines/pipeline.yaml

```diff
@@ -1,23 +1,60 @@
-pr:
```

Comment:
Why remove the pr trigger and leave the merge queue trigger? I think it should be the other way around.